<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>KBWS Application</title> 
    <meta name="keywords" lang="en" content="EMBOSS, European
    Molecular Biology Open Source Software Suite, bioinformatics, 
    EBI, European Bioinformatics Institute, Molecular Biology, 
    Computational Biology, Open Source, Software, Database" />
  </head>

<body bgcolor="#ffffff" style="background-color=#ffffff; body-background-color=#ffffff">
<link rel="stylesheet" href="../../stylesheets/EMBOSS.css" type="text/css" >

<!-- BEGIN PAGE HEADING -->
<div id="container">
<div id="header">  <a href="../index.html"> <img border="0" src="../../images/emboss.jpg" > </a> </div>
<!--#include virtual="../../header.inc" -->
<!-- END PAGE HEADING -->

<!--BEGIN MAIN CONTENT -->
<h1>KBWS Applications</h1>

<hr>



The KBWS programs are wrapper programs against KBWS SOAP server.

<p>
The KBWS versions of these programs all have the prefix "k" to
distinguish them from the original services. 

<p>

<h3><A NAME="kbws">Applications</A> in the <a
href="ftp://emboss.open-bio.org/pub/EMBOSS/">current kbws release</a></h3>

<table border cellpadding=4 bgcolor="#FFFFF0">

<tr>
<th>Program name</th>
<th>Description</th>
</tr>

<tr>
<td><a href="kdollop.html">kdollop</a></td>
<td>
Dollo and polymorphism parsimony algorithm using dollop
</td>
</tr>
<tr>
<td><a href="kkitsch.html">kkitsch</a></td>
<td>
Fitch-Margoliash method with contemporary tips using kitsch
</td>
</tr>
<tr>
<td><a href="ktrnascan_se.html">ktrnascan_se</a></td>
<td>
Finds tRNA genes using tRNAscan-SE
</td>
</tr>
<tr>
<td><a href="kgenemarkhmm.html">kgenemarkhmm</a></td>
<td>
Finds genes in prokaryotic genomes using GeneMarkhmm
</td>
</tr>
<tr>
<td><a href="kfetchdata.html">kfetchdata</a></td>
<td>
Fetch an entry in a defined format and style
</td>
</tr>
<tr>
<td><a href="kdnainvar.html">kdnainvar</a></td>
<td>
Nucleic acid sequence Invariants method using dnainvar
</td>
</tr>
<tr>
<td><a href="kdnapenny.html">kdnapenny</a></td>
<td>
Penny algorithm for DNA using dnapenny
</td>
</tr>
<tr>
<td><a href="kdnapars.html">kdnapars</a></td>
<td>
DNA parsimony algorithm using dnapars
</td>
</tr>
<tr>
<td><a href="kseqboot.html">kseqboot</a></td>
<td>
Bootstrapped sequences algorithm using seqboot
</td>
</tr>
<tr>
<td><a href="kpsort2.html">kpsort2</a></td>
<td>
Predicts protein localization for PSORT2
</td>
</tr>
<tr>
<td><a href="kprotpars.html">kprotpars</a></td>
<td>
Protein parsimony algorithm using protpars
</td>
</tr>
<tr>
<td><a href="krestml.html">krestml</a></td>
<td>
Restriction site maximum Likelihood method using restml
</td>
</tr>
<tr>
<td><a href="kweblogo.html">kweblogo</a></td>
<td>
make the generation of sequence logos using WebLogo
</td>
</tr>
<tr>
<td><a href="kcontml.html">kcontml</a></td>
<td>
Gene frequency and continuous character Maximum Likelihood using contml
</td>
</tr>
<tr>
<td><a href="kdnaml.html">kdnaml</a></td>
<td>
Estimates nucleotide phylogeny by maximum likelihood using dnaml
</td>
</tr>
<tr>
<td><a href="kneighbor.html">kneighbor</a></td>
<td>
Phylogenies from distance matrix by N-J or UPGMA method using neighbor
</td>
</tr>
<tr>
<td><a href="kdnadist.html">kdnadist</a></td>
<td>
Nucleic acid sequence Distance Matrix program using dnadist
</td>
</tr>
<tr>
<td><a href="kdolpenny.html">kdolpenny</a></td>
<td>
Penny algorithm Dollo or polymorphism using dolpenny
</td>
</tr>
<tr>
<td><a href="kdnamlk.html">kdnamlk</a></td>
<td>
Estimates nucleotide phylogeny by maximum likelihood using dnamlk
</td>
</tr>
<tr>
<td><a href="kprotdist.html">kprotdist</a></td>
<td>
Protein distance algorithm using protdist
</td>
</tr>
<tr>
<td><a href="kpsortb.html">kpsortb</a></td>
<td>
Predicts protein localization for PSORTB
</td>
</tr>
<tr>
<td><a href="kdnacomp.html">kdnacomp</a></td>
<td>
DNA compatibility algorithm using dnacomp
</td>
</tr>
<tr>
<td><a href="kkalign.html">kkalign</a></td>
<td>
Multiple sequence alignment using Kalign
</td>
</tr>
<tr>
<td><a href="kpsort.html">kpsort</a></td>
<td>
Predicts protein localization for PSORT
</td>
</tr>
<tr>
<td><a href="kglimmer.html">kglimmer</a></td>
<td>
Finding gene in prokaryotic genomes using glimmer
</td>
</tr>
<tr>
<td><a href="kcentroidfold.html">kcentroidfold</a></td>
<td>
Predicts RNA 2D structure using CentroidFold
</td>
</tr>
<tr>
<td><a href="kgendist.html">kgendist</a></td>
<td>
Compute genetic distances from gene frequencies using gendist
</td>
</tr>
<tr>
<td><a href="kfetchbatch.html">kfetchbatch</a></td>
<td>
Fetch a set of entries in a defined format and style
</td>
</tr>
<tr>
<td><a href="kclique.html">kclique</a></td>
<td>
Largest clique program using clique
</td>
</tr>
<tr>
<td><a href="kblast.html">kblast</a></td>
<td>
Search similar sequences in public repositories using BLAST
</td>
</tr>
<tr>
<td><a href="kmuscle.html">kmuscle</a></td>
<td>
public domain multiple alignment using MUSCLE
</td>
</tr>
<tr>
<td><a href="kmix.html">kmix</a></td>
<td>
Mixed parsimony algorithm using mix
</td>
</tr>
<tr>
<td><a href="kssearch.html">kssearch</a></td>
<td>
compares query sequences to databases using SSEARCH
</td>
</tr>
<tr>
<td><a href="ktcoffee.html">ktcoffee</a></td>
<td>
Multiple sequence alignment using T-COFFEE
</td>
</tr>
<tr>
<td><a href="kwolfpsort.html">kwolfpsort</a></td>
<td>
Predicts protein localization for WoLF PSORT
</td>
</tr>
<tr>
<td><a href="kmafft.html">kmafft</a></td>
<td>
Multiple sequence alignment using MAFFT
</td>
</tr>
<tr>
<td><a href="kphobius.html">kphobius</a></td>
<td>
Predicts transmembrance topology and signal peptides using Phobius
</td>
</tr>
<tr>
<td><a href="kclustalw.html">kclustalw</a></td>
<td>
Multiple sequence alignment using ClustalW 2
</td>
</tr>
<tr>
<td><a href="kfitch.html">kfitch</a></td>
<td>
Fitch-Margoliash and Least-Squares Distance Methods using fitch
</td>
</tr>
<tr>
<td><a href="kpenny.html">kpenny</a></td>
<td>
Penny algorithm, branch-and-bound using penny
</td>
</tr>
</table>

<hr>
</div>
</BODY>
</HTML>
