<!--START OF HEADER - DON'T ALTER -->

<HTML>

  <HEAD>
    <TITLE>EMBOSS: kcentroidfold</TITLE>
  </HEAD>

  <BODY BGCOLOR="#FFFFFF" text="#000000">
    <table align=center border=0 cellspacing=0 cellpadding=0>
      <tr><td valign=top>
	  <A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
	</td>
	<td align=left valign=middle>
	  <b><font size="+6">
	      kcentroidfold
	  </font></b>
      </td></tr>
    </table>
    <br>&nbsp;
    <p>
  <!--END OF HEADER-->

  <H2>Function</H2>
  Predicts RNA 2D structure using CentroidFold
  <!--
      DON'T WRITE ANYTHING HERE.
      IT IS DONE FOR YOU.
    -->

  <h2>Description</h2>
    <p>
      kcentroidfold predicts an RNA 2D structure from given RNA sequence
      utilizing CentroidFold.
      CentroidFold is based on the gamma-centroid estimator for high-dimensional
      discrete spaces.
    </p>

    <p>
      kcentroidfold utilizes CentroidFold Web Server provided by ncRNA.org.
      Original web-service is located at the following URL:
    </p>

    <p>
      <a href="http://www.ncrna.org/centroidfold/">http://www.ncrna.org/centroidfold/</a>
    </p>

    <p>
      This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,
      which contains interfaces to bioinformatics web services through a proxy
      server at Keio University. kcentroidfold is thus an interface of "runCentroidFold"
      method included in KBWS SOAP service.
    </p>
    <p>
      This method can be alternatively accessed directly from programming languages
      as SOAP web service. Please refer to the KBWS online documentations
      <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
    </p>

    <p>
      <h2>Usage</h2>

      <table width="90%">
	<tr>
	  <td bgcolor="#CCFFFF">
	    <pre>

    Here is a sample session with kcentroidfold

    % kcentroidfold
    RNA 2D structure prediction from an RNA sequence using CentroidFold
    Input (gapped) sequence(s): test.fasta
    Output image file name [kcentroidfold.png]:
    Output file [55-110.centroidfold]:
    Jobid: kbws_12547145
    ****

    % cat 55-110.centroidfold
    >M33000.1/55-110
    ACGCUGUCUGUACUUGUAUCAGUACACUGACGAGUCCCUAAAGGACGAAACAGCGC
    .(((((((((((((.(...)))))))..).(..((((.....)))))..)))))).

    % ls kcentroidfold.png
    kcentroidfold.png
	    </pre>
	  </td>
	</tr>
      </table>

      <h2>Command line arguments</h2>
      <table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
	<tr bgcolor="#FFFFCC">
	  <th align="left">Qualifier</th>
	  <th align="left">Type</th>
	  <th align="left">Description</th>
	  <th align="left">Allowed values</th>
	  <th align="left">Default</th>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <td>[-seqall]<br>(Parameter 1)</td>
	  <td>seqall</td>
	  <td>(Gapped) sequence(s) filename and optional format, or reference (input USA)</td>
	  <td>Readable sequence(s)</td>
	  <td>2</td>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <td>[-outfile]<br>(Parameter 2)</td>
	  <td>outfile</td>
	  <td>Output file name</td>
	  <td>Output file</td>
	  <td><i>&lt;*&gt;</i>.kcentroidfold</td>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <td>[-goutfile]<br>(Parameter 3)</td>
	  <td>string</td>
	  <td>Output image file name</td>
	  <td>String</td>
	  <td>kcentroid.png</td>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <th align="left" colspan=5>Additional (Optional) qualifiers</th>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <td>-engine</td>
	  <td>string</td>
	  <td>specify the inference engine</td>
	  <td>'CONTRAfold', 'McCaskill', 'pfold' or 'AUX'</td>
	  <td>CONTRAfold</td>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <td>-gamma</td>
	  <td>integer</td>
	  <td>weight of base pairs (2^-5 ~ 2^10)</td>
	  <td>Any integer value</td>
	  <td>4</td>
	</tr>

	<tr bgcolor="#FFFFCC">
	  <th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
	</tr>

	<tr>
	  <td colspan=5>(none)</td>
	</tr>
      </table>

      <h2>Input file format</h2>
      kcentroidfold can use any genome sequence data that seqret can read.

      <h2>Output file format</h2>
      kcentroidfold outputs "Seq/Fold information" and "graph result".
      "Seq/Fold information" is general CentroidFold output file.
      "graph result" is an URL about image file of predicted 2D structure.

      <h2>Data files</h2>
      None.

      <h2>Notes</h2>
      None.

      <h2>References</h2>
      <pre>
	Kengo, Sato., et al.,(2009) CentroidFold: a web server for RNA secondary
	structure prediction, Nucleic Acids Research, 37, 277-280.
	Michiaki, Hamada., et al.,(2009) Predictions of RNA secondary structure
	using generalized centroid estimators, Bioinformatics, 25(4), 465-473. 
      </pre>

      <h2>Warnings</h2>
      None.

      <h2>Diagnostic Error Messages</h2>
      None.

      <h2>Exit status</h2>
      It always exits with a status of 0.

      <h2>Known bugs</h2>
      None.

      <h2>See also</h2>
      <table border cellpadding=4 bgcolor="#FFFFF0">
	<tr>
	  <th>Program name</th>
	  <th>Description</th>
	</tr>
	<tr>
	  <td><a href="seqret.html">seqret</a></td>
	  <td>Reads and writes (returns) sequences</td>
	</tr>

	<tr>
	  <td><a href="krnafold.html">krnafold</a></td>
	  <td>Predicts secondary structure of single starnd RNA or DNA using RNAfold</td>
	</tr>
      </table>

      <h2>Author(s)</h2>
      Kazuki Oshita (cory @ g-language.org)<br />
      Institute for Advanced Biosciences, Keio University<br />
      252-8520 Japan<br />
      
      <h2>History</h2>
      2009 - Written by Kazuki Oshita

      <h2>Target users</h2>
      This program is intended to be used by everyone and everything, from
      naive users to embedded scripts.

      <h2>Comments</h2>

  </BODY>
</HTML>
