<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE>
  EMBOSS: kcontml
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
kcontml
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2>
    Function
</H2>
Gene frequency and continuous character Maximum Likelihood using contml
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<h2>Description</h2>


   kcontml is gene frequency and continuous character Maximum Likelihood
   utilizing "contml" methods provided in PHYLIP web service.
</p>

<p>   Following block is a quotation from fcontml included in PHYLIPNEW
   package.
     
     Estimates phylogenies from gene frequency data by maximum likelihood
     under a model in which all divergence is due to genetic drift in the
     absence of new mutations. Does not assume a molecular clock. An
     alternative method of analyzing this data is to compute Nei's genetic
     distance and use one of the distance matrix programs. This program can
     also do maximum likelihood analysis of continuous characters that
     evolve by a Brownian Motion model, but it assumes that the characters
     evolve at equal rates and in an uncorrelated fashion, so that it does
     not take into account the usual correlations of characters.
  
   PHYLIP web service is provided by National University of Singapore.
   Original web-service is located at the following URL:
</p>

<p>   http://bioportal.bic.nus.edu.sg/phylip/contml.html
</p>

<p>   This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,
   which contains interfaces to bioinformatics web services through a proxy
   server at Keio University. kcontml is thus an interface of "runContml"
   method included in KBWS SOAP service.
   This method can be alternatively accessed directly from programming languages
   as SOAP web service. Please refer to the KBWS online documentations
   http://soap.g-language.org/kbws/ for more information.
</p>

<p>

<h2>Usage</h2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with kcontml

% kcontml
Gene frequency and continuous character Maximum Likelihood using contml
Input file: sample.dat
Output file [55-110.kcontml]:
Jobid: kbws_12547145
***********


</pre></td></tr></table>


<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-infile]<br>(Parameter 1)</td>
<td>infile</td>
<td>data file for Contml</td>
<td>Input file</td>
<td>2</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.kcontml</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<h3>Input files for usage example</h3>
<p><h3>File: sample.dat</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
</pre>
</td></tr></table><p>

<h2>Output file format</h2>

<h3>Output files for usage example</h3>
<p><h3>File: sample.kcontml</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>5 Populations,   10 Loci

Continuous character Maximum Likelihood method version 3.57c


  +-----------------------------------------------------------
Australia
  !  
  !  +--------
African  
--3--1  
  !  !   +-
American 
  !  +---2  
  !      +----
Chinese  
  !  
  +
European 


remember: this is an unrooted tree!

Ln Likelihood =    22.00022

examined   15 trees

Between     And             Length      Approx. Confidence Limits
-------     ---             ------      ------- ---------- ------
  3       
Australia        0.59063   (     0.19034,     1.20969)
  3          1              0.00641   (    -0.01944,     0.04640)
  1       
African          0.08446   (     0.02336,     0.17896)
  1          2              0.03614   (    -0.00236,     0.09568)
  2       
American         0.02029   (    -0.00942,     0.06625)
  2       
Chinese          0.05347   (     0.00795,     0.12387)
  3       
European         0.00000   (    -0.02428,     0.03756)





</pre>
</td></tr></table><p>

<h2>Data files</h2>


   None.



<h2>Notes</h2>


   None.



<h2>References</h2>


<pre>

   Felsenstein, J.,(1995) PHYLIP(Phylogeny Interface Package), version3.57c,
      Department of Genetics, University of Washington, Seattle.

   Swofford, D., et al.,(1996) Phylogenetic interface. In Hillis, D.M.,
      Moritz, C., and Mble, B.K.(eds), Molecular Systematics 2nd Edition,
      Sinauer Associates, Mss.


</pre>


<h2>Warnings</h2>


   None.



<h2>Diagnostic Error Messages</h2>


   None.



<h2>Exit status</h2>


   It always exits with a status of 0.



<h2>Known bugs</h2>


   None.



<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="seqret.html">seqret</a></td>
<td>Reads and writes (returns) sequences</td>
</tr>

</table>


<h2>Author(s)</h2>


   Kazuki Oshita (cory @ g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan



<h2>History</h2>


   2009 - Written by Kazuki Oshita



<h2>Target users</h2>


   This program is intended to be used by everyone and everything, from
   naive users to embedded scripts.



<h2>Comments</h2>














</BODY>
</HTML>

