<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE>
  EMBOSS: kdnamlk
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
kdnamlk
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2>
    Function
</H2>
Estimates nucleotide phylogeny by maximum likelihood using dnamlk
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<h2>Description</h2>


   kdnamlk is estimates nucleotide phylogeny by maximum likelihood
   utilizing "dnamlk" methods provided in PHYLIP web service.
</p>

<p>   Following block is a quotation from fdnamlk included in PHYLIPNEW
   package.
     
     Same as DNAML but assumes a molecular clock. The use of the two
     programs together permits a likelihood ratio test of the molecular
     clock hypothesis to be made.
     Estimates phylogenies from nucleotide sequences by maximum likelihood.
     The model employed allows for unequal expected frequencies of the four
     nucleotides, for unequal rates of transitions and transversions, and
     for different (prespecified) rates of change in different categories
     of sites, and also use of a Hidden Markov model of rates, with the
     program inferring which sites have which rates. This also allows
     gamma-distribution and gamma-plus-invariant sites distributions of
     rates across sites.
  
   PHYLIP web service is provided by National University of Singapore.
   Original web-service is located at the following URL:
</p>

<p>   http://bioportal.bic.nus.edu.sg/phylip/dnamlk.html
</p>

<p>   This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,
   which contains interfaces to bioinformatics web services through a proxy
   server at Keio University. kdnamlkis thus an interface of "runDnamlk"
   method included in KBWS SOAP service.
   This method can be alternatively accessed directly from programming languages
   as SOAP web service. Please refer to the KBWS online documentations
   <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
</p>

<p>

<h2>Usage</h2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with kdnamlk

% kdnamlk
Estimates nucleotide phylogeny by maximum likelihood using dnamlk
Input (gapped) sequence(s): test.fasta
Output file [55-110.kdnamlk]:

</pre></td></tr></table>


<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-seqall]<br>(Parameter 1)</td>
<td>seqall</td>
<td>(Gapped) sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td>2</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.kdnamlk</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<h3>Input files for usage example</h3>
<p><h3>File: sample.dat</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
     4   13
Beta      AAGGUCGCCAAAC
Gamma     CAUUUCGUCACAA
Delta     GGUAUUUCGGCCU
Epsilon   GGGAUCUCGGCCC</pre>
</td></tr></table><p>

<h2>Output file format</h2>

<h3>Output files for usage example</h3>
<p><h3>File: sample.kdnamlk</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>4 Species,   15 Sites

Nucleic acid sequence
   Maximum Likelihood method with molecular clock, version 3.57c


Empirical Base Frequencies:

   A       0.21154
   C       0.30769
   G       0.25000
  T(U)     0.23077

Transition/transversion ratio =   2.000000

(Transition/transversion parameter =   1.514095)





  +-----------------------------------------------------------Beta      
  !  
--3                                                           +Epsilon   
  !                                                        +--1  
  +--------------------------------------------------------2  +Delta     
                                                           !  
                                                           +--Gamma     


Ln Likelihood =   -59.92907

 Ancestor      Node      Node Time     Length
 --------      ----      ---- ----     ------
 root            3      
   3          Beta         6.47680     6.47680
   3             2         6.19305     6.19305
   2             1         6.33023     0.13718
   1          Epsilon      6.47680     0.14657
   1          Delta        6.47680     0.14657
   2          Gamma        6.47680     0.28376





</pre>
</td></tr></table><p>

<h2>Data files</h2>


   None.



<h2>Notes</h2>


   None.



<h2>References</h2>


<pre>

   Felsenstein, J.,(1995) PHYLIP(Phylogeny Interface Package), version3.57c,
      Department of Genetics, University of Washington, Seattle.

   Swofford, D., et al.,(1996) Phylogenetic interface. In Hillis, D.M.,
      Moritz, C., and Mble, B.K.(eds), Molecular Systematics 2nd Edition,
      Sinauer Associates, Mss.


</pre>


<h2>Warnings</h2>


   None.



<h2>Diagnostic Error Messages</h2>


   None.



<h2>Exit status</h2>


   It always exits with a status of 0.



<h2>Known bugs</h2>


   None.



<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="seqret.html">seqret</a></td>
<td>Reads and writes (returns) sequences</td>
</tr>

</table>


<h2>Author(s)</h2>


   Kazuki Oshita (cory @ g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan



<h2>History</h2>


   2009 - Written by Kazuki Oshita



<h2>Target users</h2>


   This program is intended to be used by everyone and everything, from
   naive users to embedded scripts.



<h2>Comments</h2>














</BODY>
</HTML>

