<HTML>
  <HEAD>
    <TITLE>EMBOSS: kmuscle</TITLE>
  </HEAD>

  <BODY BGCOLOR="#FFFFFF" text="#000000">
    <table align=center border=0 cellspacing=0 cellpadding=0>
      <tr>
	<td valign=top>
	  <a HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
	</td>
	<td align=left valign=middle>
	  <b><font size="+6">kmuscle</font></b>
	</td>
      </tr>
    </table>

    <br />
    <H2>Function</H2>
    Multiple sequence alignment using MUSCLE

    <h2>Description</h2>

    <p>
      kmuscle calculates domain multiple alignment utilizing MUSCLE (MUltiple Sequence Comparison by Log-Expectation). MUSCLE is claimed to achieve both better average accuracy and better speed than ClustalW or T-Coffee, depending on the chosen options.
    </p>

    <p>
      kmuscle utilizes MUSCLE SOAP service provided by WUR (Wageningen UR (University & Research centre)). Original web-service is located at the following URL:
    </p>

    <p>
      <a href="http://www.bioinformatics.nl/tools/muscle.html">http://www.bioinformatics.nl/tools/muscle.html</a>
    </p>

    <p>
      This tools is a subset of Keio Bioinformatics Web Service (KBWS) package, which contains interfaces to bioinformatics web services through a proxy server at Keio University. kmuscle is thus an interface of "runMuscle" method included in KBWS SOAP service.<br />
      This method can be alternatively accessed directly from programming languages as SOAP web service. Please refer to the KBWS online documentations <a href="http://www.g-language.org/kbws/">http://soap.g-language.org/kbws/</a> for more information.
    </p>

    <h2>Usage</h2>
    <table width="90%">
      <tr>
	<td bgcolor="#CCFFFF">
	  <pre>

	    Here is a sample session with kmuscle

	    % kmuscle
	    Multiple sequence alignment using MUSCLE
	    Input (gapped) sequence(s): multiNuc.fasta
	    Output alignment format [fasta]: 
	    Output file [test0.kmuscle]: 
	    Jobid: kbws_5626576
	    **************

	    >testD
	    ------------------------------------------------------------
	    -------------------------TGATGAAGAATTG----------------------
	    ------------------------GGTACTGATGAT-----------------------G
	    AAGACTTTT----ACTTAAAA-----------------------------GGAGCGA---
	    -----TGCAAAACTTTAGTTATGT----------------GAACTATGTTTCACCATTAA
	    AAG------------TAATATCTGATCCAAGTACTGGAATAGTCA---------------
	    GTTCCAAAAAGAATAATGCTGAAA----------------------TGAAAAGTAAACAA
	    ATGTCAACTGATCAAATGACTAGTGAAAAAGAATTTGATTATTACACTGA---AACACTT
	    AAAGCATTATTAGAGAAAGAAGATAGTG-------------------CAGAATTAAATGA
	    AAATGAAAAAAAACTAGTTGAAACCATTAAGAAAGCTTACACTATTGA------------
	    ----------------AAAAGATAGTTCAA----------------------------TT
	    CGGTGAAACCAATTGGTCG----------------------------------------A
	    AAAACCAATTTCTCCCT-------------------------------------------
	    ---------------------------------------TACAACGTAGTAATTTAT---
	    ------------------CGTTATCTTGATT-------AGACTTTAAATTACACTGGT--
	    G-----ATAATAT------------------GGAACAACCGTT----GTGTGTTTTAGGG
	    ATTGAAACAA---------------------CCTGTGATGATACAGGTCTTAGTATTGTC
	    ATTGATCAAAAAATCAAGAGTAACATTGTTATCTCTTCTGCTAACTTACATGTAAAAACA
	    GGAGGAGTTGTACCTGAAATTGC----------------------AGCACGATGCCACGA
	    ACAAAATCTCTTTAAAGCAATAAGAGATTTAAATTTTGAGATAAGAGAT-----------
	    -----------TTATCTCACATTGCTTATGCATGTAATCCTGGGTT--------------
	    ------AGCAGGATGTTTACATGTGGGAGCCACTTTTGCTAGAA----------------
	    ----------------GCTTAAGTTTCTTATTAGACAAACCATTGTTACCCATCAACCAT
	    CTTTATGCGCATATCTTTTCTTGTTTAATTGATC-------------
	    >testF
	    -------------------AAGTCTTGCTGGCTTTGATACTCCTTTTAGTCCAGATAACC
	    TCCAG---------------------TATTTAGAAAAA----------------------
	    ----G--------------------AAACTGATTATGATCAGA------------ACTTT
	    AAAAGTTTT----ACTGAAAA-------------------GTTTAAAGATGAAAAAATAA
	    CTAACAACCAACTTGGTATTGT------------------TGATATCTATAACTTATTCA
	    GTGGTT---------T------TCACAAGAGTGTCAAAAGCACAG---TTGACTTAATGA
	    ACCAACTGCAAAAGCAAGTTGAAG----------------------CTGCTAATGCTATC
	    TTCCCAGTTGATGATGCTTTTGTTAAA-----------TTACCTAA-------AGTACCT
	    ACTGAATTATTTAAGTTAGTTGATGATA----------ATGTCTTTCCTAAGTTAAATCC
	    TAAGGGCTTAAATATCTCTGATAATATTGCTGCACTTTTTGAAAGATATAACCTTAAATC
	    GATTGA----------ACTTAAGAACTTTG-----------------ACTTAGCTTTCTT
	    GAGAAAAGCTGATGTA-------------------------------------ATTATCA
	    AAGACAAGGTTCGATAT-----------AACTTTGAGATGCAAATG-------------C
	    AGTTTCAAACTGTTTATGTTGGTGGAGGGAATACCGTTATTAACTTAGACTTTACTT---
	    -----TAAAAGCCCAAACCGTTAACTTTGCTAACCTCCAAGATTTACAAAACACTTTT--
	    G-----TTAAAGT----------------TGGTAATGATCTTTCCACCCAACTCTTTTGG
	    ATTCCAACTGTTAATAAATTAACTGATAATGCAGGTAATGATCTTACCCATATTGCT---
	    ---------AAAACTGTGATTGGTGAATCGTTTTTCCAAACCAATGTTAAC---TTAGCT
	    AAATCAGTTAT----------------------------------TGAATATGATAAGGT
	    TCAAC-----CATTGGTTAAACAAGCTTTTGA-------------AGAG-----------
	    -----------CGAGTTTTAACTCCTTTCAAAAAGGAAAGAGAAGCTGCTAAAAAAGCTT
	    ATGAAGAAGAACAACGTCGCTTGGAAGAGGAACGTAAGCGTCAACTAGAAGAGCTAAGGA
	    GAAGA---------------GAAGCTGAGGAGAAAAGAAAAGCTGAAGAAGCAAAACGAA
	    ATCAAGAAAA-------------------------------------

	  </pre>
	</td>
      </tr>
    </table>

    <h2>Command line arguments</h2>
    <table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
      <tr bgcolor="#FFFFCC">
	<th align="left">Qualifier</th>
	<th align="left">Type</th>
	<th align="left">Description</th>
	<th align="left">Allowed values</th>
	<th align="left">Default</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-seqall]<br>(Parameter 1)</td>
	<td>seqall</td>
	<td>(Gapped) sequence(s) filename and optional format, or reference (input USA)</td>
	<td>Readable sequence(s)</td>
	<td><b>Required</b></td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-output]<br>(Parameter 2)</td>
	<td>string</td>
	<td>Output alignment format. 'fasta'(FASTA), 'clw'(ClustalW), 'html'(HTML) or 'msf'(MSF)</td>
	<td>Any string</td>
	<td>fasta</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-outfile]<br>(Parameter 3)</td>
	<td>outfile</td>
	<td>Output file name</td>
	<td>Output file</td>
	<td><i>&lt;*&gt;</i>.kmuscle</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Additional (Optional) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-outorder</td>
	<td>string</td>
	<td>Sequence Order 'aligned' or 'input'(input order)</td>
	<td>Any string</td>
	<td>aligned</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-gapopen</td>
	<td>float</td>
	<td>Gap Open Penalty</td>
	<td>Any float value</td>
	<td>-3.00</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-gapextend</td>
	<td>float</td>
	<td>Gap Extension Penalty</td>
	<td>Any float value</td>
	<td>-0.275</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
      </tr>

      <tr>
	<td colspan=5>(none)</td>
      </tr>
    </table>

    <h2>Input file format</h2>
    kmuscle can use any multiple sequences data that seqret can read.

    <h2>Output file format</h2>
    The output from kmuscle is 7 types, and user can specify with "-output" option.

    <h2>Data files</h2>
    None.

    <h2>Notes</h2>
    None.

    <h2>References</h2>
    <pre>
      Edgar, R.C.,(2004) MUSCLE: multiple sequence alignment with high accuracy
      and high throughput, Nucleic Acids Research 32(5), 1792-1797.
    </pre>

    <h2>Warnings</h2>
    None.

    <h2>Diagnostic Error Messages</h2>
    None.

    <h2>Exit status</h2>
    It always exits with a status of 0.

    <h2>Known bugs</h2>
    None.

    <h2>See also</h2>
    <table border cellpadding=4 bgcolor="#FFFFF0">
      <tr>
	<th>Program name</th>
	<th>Description</th>
      </tr>

      <tr>
        <td><a href="emma.html">emma</a></td>
        <td>Multiple sequence alignment (ClustalW wrapper)</td>
      </tr>

      <tr>
        <td><a href="kclustalw.html">kclustalw</a></td>
        <td>Multiple sequence alignment using ClustalW 2</td>
      </tr>

      <tr>
        <td><a href="kkalign.html">kkalign</a></td>
        <td>Multiple sequence alignment using Kalign</td>
      </tr>

      <tr>
        <td><a href="kmafft.html">kmafft</a></td>
        <td>Multiple sequence alignment using MAFFT</td>
      </tr>

      <tr>
        <td><a href="ktcoffee.html">ktcoffee</a></td>
        <td>Multiple sequence alignment using T-COFFEE</td>
      </tr>

      <tr>
	<td><a href="seqret.html">seqret</a></td>
	<td>Reads and writes (returns) sequences</td>
      </tr>
    </table>

    <h2>Author(s)</h2>
    Kazuki Oshita (cory @ g-language.org)<br />
    Institute for Advanced Biosciences, Keio University<br />
    252-8520 Japan

    <h2>History</h2>
    2009 - Written by Kazuki Oshita<br />
    Nov 2010 - Change original web-based service<br />
    Mar 2013 - Update document by Kazuki Oshita

    <h2>Target users</h2>
    This program is intended to be used by everyone and everything, from naive users to embedded scripts.

    <h2>Comments</h2>
    None.

  </BODY>
</HTML>
