<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE>
  EMBOSS: kpenny
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
kpenny
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2>
    Function
</H2>
Penny algorithm, branch-and-bound using penny
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<h2>Description</h2>


   kpenny is penny algorithm, branch-and-bound
   utilizing "penny" methods provided in PHYLIP web service.
</p>

<p>   Following block is a quotation from fpenny included in PHYLIPNEW
   package.
     
     Finds all most parsimonious phylogenies for discrete-character data
     with two states, for the Wagner, Camin-Sokal, and mixed parsimony
     criteria using the branch-and-bound method of exact search. May be
     impractical (depending on the data) for more than 10-11 species.
  
   PHYLIP web service is provided by National University of Singapore.
   Original web-service is located at the following URL:
</p>

<p>   http://bioportal.bic.nus.edu.sg/phylip/penny.html
</p>

<p>   This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,
   which contains interfaces to bioinformatics web services through a proxy
   server at Keio University. kpennyis thus an interface of "runPenny"
   method included in KBWS SOAP service.
   This method can be alternatively accessed directly from programming languages
   as SOAP web service. Please refer to the KBWS online documentations
   http://soap.g-language.org/kbws/ for more information.
</p>

<p>

<h2>Usage</h2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with kpenny

% kpenny
Penny algorithm, branch-and-bound using penny
Input file: sample.dat
Output file [55-110.kpenny]:
Jobid: kbws_12547145
***********


</pre></td></tr></table>


<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-infile]<br>(Parameter 1)</td>
<td>infile</td>
<td>data file for penny</td>
<td>Input file</td>
<td>2</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.kpenny</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<h3>Input files for usage example</h3>
<p><h3>File: sample.dat</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
      7    6
Alpha1    110110
Alpha2    110110
Beta1     110000
Beta2     110000
Gamma1    100110
Delta     001001
Epsilon   001110</pre>
</td></tr></table><p>

<h2>Output file format</h2>

<h3>Output files for usage example</h3>
<p><h3>File: sample.kpenny</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>Penny algorithm, version 3.57c
 branch-and-bound to find all most parsimonious trees

Wagner parsimony method

                     


requires a total of              8.000

    3 trees in all found




  +-----------------
Alpha1   
  !  
  !        +--------
Alpha2   
--1        !  
  !  +-----4     +--
Epsilon  
  !  !     !  +--6  
  !  !     +--5  +--
Delta    
  +--2        !  
     !        +-----
Gamma1   
     !  
     !           +--
Beta2    
     +-----------3  
                 +--
Beta1    

  remember: this is an unrooted tree!




  +-----------------
Alpha1   
  !  
--1  +--------------
Alpha2   
  !  !  
  !  !           +--
Epsilon  
  +--2        +--6  
     !  +-----5  +--
Delta    
     !  !     !  
     +--4     +-----
Gamma1   
        !  
        !        +--
Beta2    
        +--------3  
                 +--
Beta1    

  remember: this is an unrooted tree!




  +-----------------
Alpha1   
  !  
  !           +-----
Alpha2   
--1  +--------2  
  !  !        !  +--
Beta2    
  !  !        +--3  
  +--4           +--
Beta1    
     !  
     !           +--
Epsilon  
     !        +--6  
     +--------5  +--
Delta    
              !  
              +-----
Gamma1   

  remember: this is an unrooted tree!




</pre>
</td></tr></table><p>

<h2>Data files</h2>


   None.



<h2>Notes</h2>


   None.



<h2>References</h2>


<pre>

   Felsenstein, J.,(1995) PHYLIP(Phylogeny Interface Package), version3.57c,
      Department of Genetics, University of Washington, Seattle.

   Swofford, D., et al.,(1996) Phylogenetic interface. In Hillis, D.M.,
      Moritz, C., and Mble, B.K.(eds), Molecular Systematics 2nd Edition,
      Sinauer Associates, Mss.


</pre>


<h2>Warnings</h2>


   None.



<h2>Diagnostic Error Messages</h2>


   None.



<h2>Exit status</h2>


   It always exits with a status of 0.



<h2>Known bugs</h2>


   None.



<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="seqret.html">seqret</a></td>
<td>Reads and writes (returns) sequences</td>
</tr>

</table>


<h2>Author(s)</h2>


   Kazuki Oshita (cory @ g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan



<h2>History</h2>


   2009 - Written by Kazuki Oshita



<h2>Target users</h2>


   This program is intended to be used by everyone and everything, from
   naive users to embedded scripts.



<h2>Comments</h2>














</BODY>
</HTML>

