<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE>
  EMBOSS: kprotdist
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
kprotdist
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2>
    Function
</H2>
Protein distance algorithm using protdist
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<h2>Description</h2>


   kprotdist is protein distance algorithm
   utilizing "protdist" methods provided in PHYLIP web service.
</p>

<p>   Following block is a quotation from fprotdist included in PHYLIPNEW
   package.
     
     Computes a distance measure for protein sequences, using maximum
     likelihood estimates based on the Dayhoff PAM matrix, the JTT matrix
     model, the PBM model, Kimura's 1983 approximation to these, or a model
     based on the genetic code plus a constraint on changing to a different
     category of amino acid. The distances can also be corrected for
     gamma-distributed and gamma-plus-invariant-sites-distributed rates of
     change in different sites. Rates of evolution can vary among sites in
     a prespecified way, and also according to a Hidden Markov model. The
     program can also make a table of percentage similarity among
     sequences. The distances can be used in the distance matrix programs.
  
   PHYLIP web service is provided by National University of Singapore.
   Original web-service is located at the following URL:
</p>

<p>   http://bioportal.bic.nus.edu.sg/phylip/protdist.html
</p>

<p>   This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,
   which contains interfaces to bioinformatics web services through a proxy
   server at Keio University. kprotdist is thus an interface of "runProtdist"
   method included in KBWS SOAP service.
   This method can be alternatively accessed directly from programming languages
   as SOAP web service. Please refer to the KBWS online documentations
   <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
</p>

<p>

<h2>Usage</h2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with kprotdist

% kprotdist
Protein distance algorithm using protdist
Input (gapped) sequence(s): test.fasta
Output file [55-110.kprotdist]:

</pre></td></tr></table>


<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-seqall]<br>(Parameter 1)</td>
<td>seqall</td>
<td>(Gapped) sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td>2</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.kprotdist</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<h3>Input files for usage example</h3>
<p><h3>File: sample.dat</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
       5    10
Alpha     ABCDEFGHIK
Beta      AB--EFGHIK
Gamma     ?BCDSFG.??
Delta     CIKDEFGHIK
Epsilon   DIKDEFGHIK</pre>
</td></tr></table><p>

<h2>Output file format</h2>

<h3>Output files for usage example</h3>
<p><h3>File: sample.kprotdist</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>5
Alpha          0.00000  0.00000  0.60114  0.56309  0.24648
Beta           0.00000  0.00000  0.36650  0.33655  0.41348
Delta          0.60114  0.36650  0.00000  0.23258  1.63793
Epsilon        0.56309  0.33655  0.23258  0.00000  1.63793
Gamma          0.24648  0.41348  1.63793  1.63793  0.00000



</pre>
</td></tr></table><p>

<h2>Data files</h2>


   None.



<h2>Notes</h2>


   None.



<h2>References</h2>


<pre>

   Felsenstein, J.,(1995) PHYLIP(Phylogeny Interface Package), version3.57c,
      Department of Genetics, University of Washington, Seattle.

   Swofford, D., et al.,(1996) Phylogenetic interface. In Hillis, D.M.,
      Moritz, C., and Mble, B.K.(eds), Molecular Systematics 2nd Edition,
      Sinauer Associates, Mss.


</pre>


<h2>Warnings</h2>


   None.



<h2>Diagnostic Error Messages</h2>


   None.



<h2>Exit status</h2>


   It always exits with a status of 0.



<h2>Known bugs</h2>


   None.



<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="seqret.html">seqret</a></td>
<td>Reads and writes (returns) sequences</td>
</tr>

</table>


<h2>Author(s)</h2>


   Kazuki Oshita (cory @ g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan



<h2>History</h2>


   2009 - Written by Kazuki Oshita



<h2>Target users</h2>


   This program is intended to be used by everyone and everything, from
   naive users to embedded scripts.



<h2>Comments</h2>














</BODY>
</HTML>

