<!--START OF HEADER - DON'T ALTER -->

<HTML>
  <HEAD>
    <TITLE>EMBOSS: kpsort</TITLE>
  </HEAD>

  <BODY BGCOLOR="#FFFFFF" text="#000000">
    <table align=center border=0 cellspacing=0 cellpadding=0>
      <tr>
	<td valign=top>
	  <a HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
	</td>

	<td align=left valign=middle>
	  <b><font size="+6">kpsort</font></b>
	</td>
      </tr>
    </table>

    <br />

    <H2>Function</H2>
    Predicts protein localization for PSORT

    <h2>Description</h2>
    <p>
      kpsort predicts protein localization sites utilizing PSORT.
    </p>

    <p>
      PSORT analyzes the input sequence by applying the stored rules for various sequence features of known protein sorting signals and reports the possiblity for the input protein to be localized at each candidate site with additional information.
    </p>

    <p>
      kpsort utilizes PSORT WWW Server provided by Tokyo University. Original web-service is located at the following URL:
    </p>

    <p>
      <a href="http://psort.ims.u-tokyo.ac.jp/form.html">http://psort.ims.u-tokyo.ac.jp/form.html</a>
    </p>

    <p>
      This tools is a subset of Keio Bioinformatics Web Service (KBWS) package,  which contains interfaces to bioinformatics web services through a proxy server at Keio University. kpsort is thus an interface of "runPsort" method included in KBWS SOAP service.<br />
      This method can be alternatively accessed directly from programming languages as SOAP web service. Please refer to the KBWS online documentations <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
    </p>

    <h2>Usage</h2>

    <table width="90%">
      <tr>
	<td bgcolor="#CCFFFF">
	  <pre>

  Here is a sample session with kpsort

% kpsort
prediction protein localization sets for bacterial/plant sequences
Input protein sequence(s): swissprot:FOXP2_HUMAN
Output file [foxp2_human.kpsort]:

Nakai server 

Query Information

ORIGIN animal
BEGIN
>MYSEQ
MMQESATETI SNSSMNQNGM STLSSQLDAG SRDGRSSGDT SSEVSTVELL
HLQQQQALQA ARQLLLQQQT SGLKSPKSSD KQRPLQVPVS VAMMTPQVIT
PQQMQQILQQ QVLSPQQLQA LLQQQQAVML QQQQLQEFYK KQQEQLHLQL
LQQQQQQQQQ QQQQQQQQQQ QQQQQQQQQQ QQQQQQQQQQ QHPGKQAKEQ
QQQQQQQQQL AAQQLVFQQQ LLQMQQLQQQ QHLLSLQRQG LISIPPGQAA
LPVQSLPQAG LSPAEIQQLW KEVTGVHSME DNGIKHGGLD LTTNNSSSTT
SSNTSKASPP ITHHSIVNGQ SSVLSARRDS SSHEETGASH TLYGHGVCKW
PGCESICEDF GQFLKHLNNE HALDDRSTAQ CRVQMQVVQQ LEIQLSKERE
RLQAMMTHLH MRPSEPKPSP KPLNLVSSVT MSKNMLETSP QSLPQTPTTP
TAPVTPITQG PSVITPASVP NVGAIRRRHS DKYNIPMSSE IAPNYEFYKN
ADVRPPFTYA TLIRQAIMES SDRQLTLNEI YSWFTRTFAY FRRNAATWKN
AVRHNLSLHK CFVRVENVKG AVWTVDEVEY QKRRSQKITG SPTLVKNIPT
SLGYGAALNA SLQAALAESS LPLLSNPGLI NNASSGLLQA VHEDLNGSLD
HIDSNGNSSP GCSPQPHIHS IHVKEEPVIA EDEDCPMSLV TTANHSPELE
DDREIEEEPL SEDLE


Result Information

PSORT --- Prediction of Protein Localization Sites
version 6.4(WWW)
MYSEQ          715 Residues
Species classification: 4

*** Reasoning Step: 1

Preliminary Calculation of ALOM (threshold: 0.5)
count: 0
McG: Examining signal sequence (McGeoch)
Length  of  UR:   19
Peak Value of UR:  -0.16
Net Charge of CR: -2
Discriminant Score:    -14.88
GvH: Examining signal sequence (von Heijne)
Signal Score (-3.5): -2.15
Possible cleavage site: 60
>>> Seems to have no N-terminal signal seq.
Amino Acid Composition of Predicted Mature Form:
calculated from 1
ALOM new cnt: 0 ** thrshld changed to -2
Cleavable signal was detected in ALOM?: 0B
ALOM: finding transmembrane regions (Klein et al.)
count: 0  value:   5.41  threshold: -2.0
PERIPHERAL  Likelihood =  5.41
modified ALOM score:  -1.98
( 3) or uncleavable?
Gavel: Examining the boundary of mitochondrial targeting seq.
motif at: 3
Uncleavable? Ipos set to: 13
Discrimination of mitochondrial target seq.: 
negative (-1.91)

*** Reasoning Step: 2

KDEL   Count: 0
Checking apolar signal for intramitochondrial sorting
Mitochondrial matrix?  Score:  0.10
SKL motif (signal for peroxisomal protein): 
pos: -1(715), count: 0   
Amino Acid Composition Tendency for Peroxisome:  -5.66
Peroxisomal proteins?   Status: negative
Amino Acid Composition tendency for lysosomal proteins
score:  0.33  Status: notclr
Modified score for lysosome: 0.183
Checking the amount of Basic Residues (nucleus)
Checking the 4 residue pattern for Nuclear Targeting
Found: pos: 475 (3) RRRH
Checking the 7 residue pattern for Nuclear Targeting
Checking the Robbins & Dingwall consensus (nucleus)
Checking the RNA binding motif (nucleus or cytoplasm)
nuc modified.   Score:  0.60
Nuclear Signal   Status: notclr ( 0.30)
Checking CaaX motif..
Checking N-myristoylation..
Checking CaaX motif..

----- Final Results -----

nucleus --- Certainty= 0.300(Affirmative) 
lysosome (lumen) --- Certainty= 0.183(Affirmative) 
mitochondrial matrix space --- Certainty= 0.100(Affirmative) 
endoplasmic reticulum (membrane) --- Certainty= 0.000(Not Clear) 


----- The End -----

	  </pre>
	</td>
      </tr>
    </table>


    <h2>Command line arguments</h2>
    <table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
      <tr bgcolor="#FFFFCC">
	<th align="left">Qualifier</th>
	<th align="left">Type</th>
	<th align="left">Description</th>
	<th align="left">Allowed values</th>
	<th align="left">Default</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-seqall]<br>(Parameter 1)</td>
	<td>seqall</td>
	<td>Protein sequence(s) filename and optional format, or reference (input USA)</td>
	<td>Readable sequence(s)</td>
	<td><b>Required</b></td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-outfile]<br>(Parameter 2)</td>
	<td>outfile</td>
	<td>Output file name</td>
	<td>Output file</td>
	<td><i>&lt;*&gt;</i>.kpsort</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Additional (Optional) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-org</td>
	<td>string</td>
	<td>Source for input sequence organisms. 'Gram-positive bacterium', 'Gram-negative bacterium', 'yeast', 'animal' or 'plant'</td>
	<td>Any string</td>
	<td>animal</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-title</td>
	<td>string</td>
	<td>title of your job</td>
	<td>Any string</td>
	<td>MYSEQ</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
      </tr>
      
      <tr>
	<td colspan=5>(none)</td>
      </tr>
    </table>


    <h2>Input file format</h2>
      kpsort can use any amino acid sequence data that seqret can read.

      <h2>Output file format</h2>
      The output from kpsort is general PSORT output format.

      <h2>Data files</h2>
      None.

      <h2>Notes</h2>
      None.

      <h2>References</h2>
      <pre>
	Nakai, K., and Kanehisa, M.,(1991) Expert system for predicting protein
	localization sites in gram-negative bacteria., Proteins, 11(2), 95-110.
      </pre>

      <h2>Warnings</h2>
      None.

      <h2>Diagnostic Error Messages</h2>
      None.

      <h2>Exit status</h2>
      It always exits with a status of 0.

      <h2>Known bugs</h2>
      None.

      <h2>See also</h2>
      <table border cellpadding=4 bgcolor="#FFFFF0">
	<tr>
	  <th>Program name</th>
	  <th>Description</th>
	</tr>

	<tr>
          <td><a href="kpsort2.html">kpsort2</a></td>
          <td>Predicts protein localization for PSORT2</td>
        </tr>

        <tr>
          <td><a href="kpsortb.html">kpsortb</a></td>
          <td>Predicts protein localization for PSORT B</td>
        </tr>

        <tr>
          <td><a href="kwolfpsort.html">kwolfpsort</a></td>
          <td>Predicts protein localization for WoLF PSORT</td>
        </tr>

	<tr>
	  <td><a href="seqret.html">seqret</a></td>
	  <td>Reads and writes (returns) sequences</td>
	</tr>
      </table>

      <h2>Author(s)</h2>
      Kazuki Oshita (cory @ g-language.org)<br />
      Institute for Advanced Biosciences, Keio University<br />
      252-8520 Japan

      <h2>History</h2>
      2009 - Written by Kazuki Oshita<br />
      Mar 2013 - Update document by Kazuki Oshita

      <h2>Target users</h2>
      This program is intended to be used by everyone and everything, from naive users to embedded scripts.

      <h2>Comments</h2>
      None.

  </BODY>
</HTML>

