<HTML>
  <HEAD>
    <TITLE>EMBOSS: kshowblastdb</TITLE>
  </HEAD>

  <BODY BGCOLOR="#FFFFFF" text="#000000">
    <table align=center border=0 cellspacing=0 cellpadding=0>
      <tr>
	<td valign=top>
	  <a HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
	</td>
	<td align=left valign=middle>
	  <b><font size="+6">kshowblastdb</font></b>
	</td>
      </tr>
    </table>

    <br />

    <H2>Function</H2>
    Displays information on databases for kblast


    <h2>Description</h2>

    <p>
      kshowblastdb displays information of kblast databases just as showdb.<br />
      This tools is a subset of Keio Bioinformatics Web Service (KBWS) package, which contains interfaces to bioinformatics web services through a proxy server at Keio University. kblast is an interface of "showBlastDB" method included in KBWS SOAP service. <br />
      This method can be alternatively accessed directly from programming languages as SOAP web service. Please refer to the KBWS online documentations <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
    </p>

    <h2>Usage</h2>
    <table width="90%">
      <tr>
	<td bgcolor="#CCFFFF">
	  <pre>

   Here is a sample session with kshowblastdb

% kshowblastdb
Displays information on databases for kblast
# Name               Type       Server  Comment
# ================== ========== ======= =======
DDBJ                 Nucleotide DDBJ    DDBJ Release and New data includes all division
DDBJ_EXEST           Nucleotide DDBJ    DDBJ Release and New data except EST division
DDBJNEW              Nucleotide DDBJ    DDBJ New data includes all division
DDBJNEW_EXEST        Nucleotide DDBJ    DDBJ New data except EST division
ddbjbct              Nucleotide DDBJ    DDBJ Bacteria division
ddbjenv              Nucleotide DDBJ    DDBJ Env division
ddbjest              Nucleotide DDBJ    DDBJ EST division
ddbjgss              Nucleotide DDBJ    DDBJ Genome Survey Sequences division
ddbjhtc              Nucleotide DDBJ    DDBJ High Throughput cDNAs division
ddbjhtg              Nucleotide DDBJ    DDBJ High Throughput Genomic sequencing division
ddbjhum              Nucleotide DDBJ    DDBJ human division
ddbjinv              Nucleotide DDBJ    DDBJ invertebrates division
ddbjmam              Nucleotide DDBJ    DDBJ other mammals division
ddbjpat              Nucleotide DDBJ    DDBJ patens division
ddbjphg              Nucleotide DDBJ    DDBJ phages division
ddbjpln              Nucleotide DDBJ    DDBJ plants, fugi and algae division
ddbjpri              Nucleotide DDBJ    DDBJ primates division

( omitting... )

repeat/repeat_7719   Nucleotide NCBI    repeat_7719
repeat/repeat_7955   Nucleotide NCBI    repeat_7955
repeat/repeat_9606   Nucleotide NCBI    repeat_9606
repeat/repeat_9989   Nucleotide NCBI    repeat_9989
DAD                  Protein    DDBJ    DAD data
PDB                  Protein    DDBJ    PDB data
PDBSH                Protein    DDBJ    PDB data (identical to 'PDB')
PRF                  Protein    DDBJ    PRF data
PROTEIN              Protein    DDBJ    PROTEIN data
SWISS                Protein    DDBJ    SWISSPROT data
WORMPEP              Protein    DDBJ    WORM PEPTIDE data
UNIPROT              Protein    DDBJ    UNIPROT data
TREMBL               Protein    DDBJ    TREMBL data
uniprot              Protein    EBI     UniProt
swissprot            Protein    EBI     UniProtKB/Swiss-Prot
uniref100            Protein    EBI     UniProt
uniref90             Protein    EBI     UniProt
uniref50             Protein    EBI     UniProt
uniparc              Protein    EBI     UniProt
ipi                  Protein    EBI     International
pdb                  Protein    EBI     Protein
sgt                  Protein    EBI     Structural
intact               Protein    EBI     IntAct
imgthlap             Protein    EBI     IMGT/HLA
epop                 Protein    EBI     EPO
jpop                 Protein    EBI     JPO
kpop                 Protein    EBI     KIPO
uspop                Protein    EBI     USPTO
nr                   Protein    NCBI    (nr, protein)
refseq_protein       Protein    NCBI    (refseq protein)
swissprot            Protein    NCBI    (swissprot protein)
pataa                Protein    NCBI    (pat protein)
pdbaa                Protein    NCBI    (pdb protein)
month.pataa          Protein    NCBI    (month pat protein)
monthaa              Protein    NCBI    (month protein)
alu                  Protein    NCBI    (alu repeat protein)
env_nr               Protein    NCBI    (env protein)

	  </pre>
	</td>
      </tr>
    </table>

    <h2>Command line arguments</h2>
    <table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
      <tr bgcolor="#FFFFCC">
	<th align="left">Qualifier</th>
	<th align="left">Type</th>
	<th align="left">Description</th>
	<th align="left">Allowed values</th>
	<th align="left">Default</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
      </tr>

      <tr>
	<td colspan=5>(none)</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Additional (Optional) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-[no]nucleotide</td>
	<td>boolean</td>
	<td>Displays information on nucleotide databases</td>
	<td>Boolean value Yes/No</td>
	<td>Yes</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-[no]protein</td>
	<td>boolean</td>
	<td>Displays information on protein databases</td>
	<td>Boolean value Yes/No</td>
	<td>Yes</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-outfile</td>
	<td>outfile</td>
	<td>Database list for kblast</td>
	<td>Output file name</td>
	<td>stdout</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
      </tr>

      <tr>
	<td colspan=5>(none)</td>
      </tr>
    </table>

    <h2>Input file format</h2>
    None.

    <h2>Output file format</h2>
    kshowblastdb


    <h2>Data files</h2>
    None.

    <h2>Notes</h2>
    None.

    <h2>References</h2>
    <pre>
      Altschul, S.F., Gish, W., Miller, W., Myers, E.W. and Lipman, D.J. (1990) Basic
      local alignment search tool, J Mol Biol, 215, 403-410.

      Altschul S.F., et al.,(1997) Gapped BLAST and PSI-BLAST: a new generation
      of protein database search programs, Nucleic Acids Research, 25(17), 3389-3402.

      McWilliam H., et al.,(2009) Web services at the European Bioinformatics Institute
      Nucleic Acids Research Web Server Issue 2009, 37, W6-W10.
    </pre>

    <h2>Warnings</h2>
    None.


    <h2>Diagnostic Error Messages</h2>
    None.

    <h2>Exit status</h2>
    It always exits with a status of 0.

    <h2>Known bugs</h2>
    None.

    <h2>See also</h2>
    <table border cellpadding=4 bgcolor="#FFFFF0">
      <tr>
        <th>Program name</th>
        <th>Description</th>
      </tr>

      <tr>
        <td><a href="kblast.html">kbkast</a></td>
        <td>Search similar sequences in public repositories using BLAST</td>
      </tr>

      <tr>
        <td><a href="showdb.html">showdb</a></td>
        <td>Displays information on configured databases</td>
      </tr>
    </table>

    <h2>Author(s)</h2>
    Kazuki Oshita (cory @ g-language.org)<br />
    Institute for Advanced Biosciences, Keio University<br />
    252-8520 Japan
    
    <h2>History</h2>
    2010 - Written by Kazuki Oshita<br />
    Mar 2013 - Update document by Kazuki Oshita

    <h2>Target users</h2>
    This program is intended to be used by everyone and everything, from naive users to embedded scripts.
      
    <h2>Comments</h2>
    None.

  </BODY>
</HTML>
