<HTML>
  <HEAD>
    <TITLE>EMBOSS: ktrnascan_se</TITLE>
  </HEAD>

  <BODY BGCOLOR="#FFFFFF" text="#000000">
    <table align=center border=0 cellspacing=0 cellpadding=0>
      <tr>
	<td valign=top>
	  <a HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
	</td>
	<td align=left valign=middle>
	  <b><font size="+6">ktrnascan_se</font></b>
	</td>
      </tr>
    </table>

    <br />

    <H2>Function</H2>
    Finds tRNA genes using tRNAscan-SE

    <h2>Description</h2>

    <p>
      ktrnascan_se detects transfer RNA genes in DNA sequence utilizingtRNAscan-SE. tRNAscan-SE identifies 99â€“100% of transfer RNA genes in DNA sequence while giving less than one false positive per 15 gigabases.
    </p>

    <p>
      ktrnascan_se utilizes tRNAscan-SE server provided by Lowe Lab. Original web-service is located at the following URL:
    </p>

    <p>
      <a href="http://lowelab.ucsc.edu/tRNAscan-SE/">http://lowelab.ucsc.edu/tRNAscan-SE/</a>
    </p>

    <p>
      This tools is a subset of Keio Bioinformatics Web Service (KBWS) package, which contains interfaces to bioinformatics web services through a proxy server at Keio University. ktrnascan_se is thus an interface of "runtRNAscan" method included in KBWS SOAP service.<br />
      This method can be alternatively accessed directly from programming languages as SOAP web service. Please refer to the KBWS online documentations <a href="http://www.g-language.org/kbws/">http://www.g-language.org/kbws/</a> for more information.
    </p>

    <h2>Usage</h2>

    <table width="90%">
      <tr>
	<td bgcolor="#CCFFFF">
	  <pre>

	    Here is a sample session with ktrnascan_se

	    % ktrnascan_se
	    Finds tRNA genes using tRNAscan-SE
	    Input (gapped) sequence(s): refseq:NC_000908
	    Output file [nc_000908.ktrnascan_se]: 
	    Jobid: kbws_4760102
	    *****


	    tRNAscan-SE v.1.21 (Oct 2000) - scan sequences for transfer RNAs

	    Please cite:
	    Lowe, T.M. & Eddy, S.R. (1997) "tRNAscan-SE: A program for
	    improved detection of transfer RNA genes in genomic sequence"
	    Nucl. Acids Res. 25: 955-964.

	    This program uses a modified, optimized version of tRNAscan v1.3
	    (Fichant & Burks, J. Mol. Biol. 1991, 220: 659-671),
	    a new implementation of a multistep weight matrix algorithm
	    for identification of eukaryotic tRNA promoter regions
	    (Pavesi et al., Nucl. Acids Res. 1994, 22: 1247-1256),
	    as well as the RNA covariance analysis package Cove v.2.4.2
	    (Eddy & Durbin, Nucl. Acids Res. 1994, 22: 2079-2088).

	    Cove-predicted tRNA secondary structures rendered by 
	    NAVIEW (c) 1988 Robert E. Bruccoleri.



	    Results

	    Sequence        Your-seq
	    Subsequence     Your-seq.t1 257160 257233

	    Sequence        Your-seq.t1
	    Source          Your-seq
	    Brief_identification tRNA-Cys
	    Transcript tRNA "TGC Cys C"
	    Score tRNAscan-SE-1.21 56.59

	    Sequence        Your-seq
	    Subsequence     Your-seq.t2 257271 257344

	    Sequence        Your-seq.t2
	    Source          Your-seq
	    Brief_identification tRNA-Pro
	    Transcript tRNA "CCA Pro P"
	    Score tRNAscan-SE-1.21 79.30

	    Sequence        Your-seq
	    Subsequence     Your-seq.t3 257351 257424

	    Sequence        Your-seq.t3
	    Source          Your-seq
	    Brief_identification tRNA-Met
	    Transcript tRNA "ATG Met M"
	    Score tRNAscan-SE-1.21 76.46

	    Sequence        Your-seq
	    Subsequence     Your-seq.t4 257447 257520

	    Sequence        Your-seq.t4
	    Source          Your-seq
	    Brief_identification tRNA-Met
	    Transcript tRNA "ATG Met M"
	    Score tRNAscan-SE-1.21 75.25

	    Sequence        Your-seq
	    Subsequence     Your-seq.t5 257561 257649
	    Sequence        Your-seq.t5
	    Source          Your-seq
	    Brief_identification tRNA-Ser
	    Transcript tRNA "TCA Ser S"
	    Score tRNAscan-SE-1.21 59.71

	    Sequence        Your-seq
	    Subsequence     Your-seq.t6 257666 257739

	    Sequence        Your-seq.t6
	    Source          Your-seq
	    Brief_identification tRNA-Met
	    Transcript tRNA "ATG Met M"
	    Score tRNAscan-SE-1.21 71.85

	    Sequence        Your-seq
	    Subsequence     Your-seq.t7 257744 257814

	    Sequence        Your-seq.t7
	    Source          Your-seq
	    Brief_identification tRNA-Asp
	    Transcript tRNA "GAC Asp D"
	    Score tRNAscan-SE-1.21 58.26

	    Sequence        Your-seq
	    Subsequence     Your-seq.t8 257820 257892

	    Sequence        Your-seq.t8
	    Source          Your-seq
	    Brief_identification tRNA-Phe
	    Transcript tRNA "TTC Phe F"
	    Score tRNAscan-SE-1.21 74.07

	    Sequence        Your-seq
	    Subsequence     Your-seq.t9 266425 266498

	    Sequence        Your-seq.t9
	    Source          Your-seq
	    Brief_identification tRNA-Arg
	    Transcript tRNA "CGC Arg R"
	    Score tRNAscan-SE-1.21 67.11

	    Sequence        Your-seq
	    Subsequence     Your-seq.t10 306619 306690

	    Sequence        Your-seq.t10
	    Source          Your-seq
	    Brief_identification tRNA-Arg
	    Transcript tRNA "AGG Arg R"
	    Score tRNAscan-SE-1.21 56.90



	    Run Statistics:

	    Started: Mon Jan  4 23:05:56 PST 2010

	    ------------------------------------------------------------
	    Search Mode:                 General
	    Searching with:              tRNAscan + EufindtRNA -> Cove
	    Covariance model:            TRNA2.cm
	    tRNAscan parameters:         Strict
	    EufindtRNA parameters:       Relaxed (Int Cutoff= -32.1)

	    Pseudogene checking disabled
	    Reporting HMM/2' structure score breakdown
	    ------------------------------------------------------------

	    First-pass (tRNAscan/EufindtRNA) Stats:
	    ---------------
	    Sequences read:         1
	    Seqs w/at least 1 hit:  1
	    Bases read:             580078 (x2 for both strands)
	    Bases in tRNAs:         3088
	    tRNAs predicted:        39
	    Av. tRNA length:        79
	    Script CPU time:        0.08 s
	    Scan CPU time:          0.39 s
	    Scan speed:             2974.8 Kbp/sec

	    First pass search(es) ended: Mon Jan  4 23:05:57 PST 2010

	    Cove Stats:
	    -----------
	    Candidate tRNAs read:     39
	    Cove-confirmed tRNAs:     36
	    Bases scanned by covels:  3634
	    % seq scanned by covels:  0.3 %
	    Script CPU time:          0.09 s
	    Cove CPU time:            5.84 s
	    Scan speed:               622.3 bp/sec

	    Cove analysis of tRNAs ended: Mon Jan  4 23:06:03 PST 2010

	    Summary
	    --------
	    Overall scan speed: 181274.4 bp/sec

	    tRNAs decoding Standard 20 AA:              35
	    Selenocysteine tRNAs (TCA):                 1
	    Possible suppressor tRNAs (CTA,TTA):        0
	    tRNAs with undetermined/unknown isotypes:   0
	    Predicted pseudogenes:                      0
            -------
	    Total tRNAs:                                36

	    tRNAs with introns:             0

	    |

	    Isotype / Anticodon Counts:

	    Ala   : 1         AGC:         GGC:         CGC:         TGC:       
	    Gly   : 2         ACC:         GCC:         CCC:         TCC:       
	    Pro   : 1         AGG:         GGG:         CGG:         TGG:       
	    Thr   : 3         AGT:         GGT:         CGT:         TGT:       
	    Val   : 1         AAC:         GAC:         CAC:         TAC:       
	    Ser   : 4         AGA:         GGA:         CGA:         TGA:         ACT:         GCT:       
	    Arg   : 4         ACG:         GCG:         CCG:         TCG:         CCT:         TCT:       
	    Leu   : 4         AAG:         GAG:         CAG:         TAG:         CAA:         TAA:       
	    Phe   : 1         AAA:         GAA:                                 
	    Asn   : 1         ATT:         GTT:                                 
	    Lys   : 2                                   CTT:         TTT:       
	    Asp   : 1         ATC:         GTC:                                 
	    Glu   : 1                                   CTC:         TTC:       
	    His   : 1         ATG:         GTG:                                 
	    Gln   : 1                                   CTG:         TTG:       
	    Ile   : 1         AAT:         GAT:                      TAT:       
	    Met   : 3                                   CAT:                    
	    Tyr   : 1         ATA:         GTA:                                 
	    Supres: 0                                   CTA:         TTA:       
	    Cys   : 1         ACA:         GCA:                                 
	    Trp   : 1                                   CCA:                    
	    SelCys: 1                                                TCA:       



	    Predicted tRNA Secondary Structures:
	    Your-seq.trna1 (257160-257233)  Length: 74 bp
	    Type: Cys       Codon: TGC at 36-38 (257195-257197)     Score: 56.59
	    HMM Sc=22.52    Sec struct Sc=34.07
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |   
	    Seq: GGTGCCTTAGCCAAGTGGActcAAGGCCTGGAGCTGCAACCTCCATAtCGTCAGTTCGAATCTGACAGGCACCT
	    Str: >>>>>>>..>>>............>>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna2 (257271-257344)  Length: 74 bp
            Type: Pro       Codon: CCA at 35-37 (257305-257307)     Score: 79.30
            HMM Sc=55.47    Sec struct Sc=23.83
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |   
            Seq: CGGGAAGTAGCTTAGTTtGGTAGAGCACTTGGTTTGGGACCAAGGGGtCGCAGGTTCAAATCCTGTCTTCCCGA
            Str: >>>>>>>..>>>>.........>>>>.......>>>>.......<<<<<<<<<<<<.

	    Your-seq.trna3 (257351-257424)  Length: 74 bp
	    Type: Met       Codon: ATG at 35-37 (257385-257387)     Score: 76.46
	    HMM Sc=43.95    Sec struct Sc=32.51
         	    *    |    *    |    *    |    *    |    *    |    *    |    *    |   
	    Seq: GGCTGGATACCTCAGTTGGTtAGAGGGCCCGGTTCATACCCGGGTTGtCGTGAGTTCGAGTCTCACTCCAGCCA
	    Str: >>>>>>>..>>>>.........>>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna4 (257447-257520)  Length: 74 bp
            Type: Met       Codon: ATG at 35-37 (257481-257483)     Score: 75.25
            HMM Sc=53.91    Sec struct Sc=21.34
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |   
            Seq: GGATCTATAGCTCAATTGGTtAGAGCCCCCGACTCATAATCGGTCTGtTACAGGTTCAAGTCCTGTTAGATCCA
            Str: >>>>>>>..>>>>.........>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna5 (257561-257649)  Length: 89 bp
            Type: Ser       Codon: TCA at 35-37 (257595-257597)     Score: 59.71
            HMM Sc=24.23    Sec struct Sc=35.48
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |    *    |    *   
            Seq: GGAGACTTACCCAAGCGGCtgAAGGGTTCGGTCTTGAAAACCGAGAgGTGCTTTATAAGCACGCGAGGGTTCGAATCCCTCAGTCTCCG
            Str: >>>>>>>..>>>...........>>>>.......>>>>>...>>>>.......<<<<<<<<<<<<.

            Your-seq.trna6 (257666-257739)  Length: 74 bp
            Type: Met       Codon: ATG at 35-37 (257700-257702)     Score: 71.85
            HMM Sc=46.39    Sec struct Sc=25.46
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |   
            Seq: CGCGGGATAGAGCAGTTGGTcAGCTCGTCAGGCTCATAATCTGAAGGtCGAGGGTTCAAATCCCTCTCCCGCAA
            Str: .>>>>>>..>>>>.........>>>>.......>>>>.......<<<<<<<<<<<..

            Your-seq.trna7 (257744-257814)  Length: 71 bp
            Type: Asp       Codon: GAC at 33-35 (257776-257778)     Score: 58.26
            HMM Sc=34.50    Sec struct Sc=23.76
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |
            Seq: GGTTCCATGGTGTAGTGATAACATATCTCCCTGTCACGGAGGGGtTGCGGGTTTGATTCCCGTTGGAACCG
            Str: >>>>>>>..>>>>.......>>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna8 (257820-257892)  Length: 73 bp
            Type: Phe       Codon: TTC at 34-36 (257853-257855)     Score: 74.07
            HMM Sc=46.53    Sec struct Sc=27.54
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |  
            Seq: GGTCTTGTAGCTCAGTCGGTAGAGCAACGGTCTGAAGAACCGTGTGtCGGCAGTTCGATTCTGCCCGAGACCA
            Str: >>>>>>>..>>>>........>>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna9 (266425-266498)  Length: 74 bp
            Type: Arg       Codon: CGC at 35-37 (266459-266461)     Score: 67.11
            HMM Sc=43.38    Sec struct Sc=23.73
                     *    |    *    |    *    |    *    |    *    |    *    |    *    |   
            Seq: GTCATCATAGCTCAATAGGAcAGAGTATCAGCTTGCGGAGCTGAGGGtTACAGGTTCGATTCCTGTTGGTGACG
            Str: >>>>>>>..>>>>.........>>>>.......>>>>.......<<<<<<<<<<<<.

            Your-seq.trna10 (306619-306690) Length: 72 bp
            Type: Arg       Codon: AGG at 34-36 (306652-306654)     Score: 56.90
            HMM Sc=22.17    Sec struct Sc=34.73
                     *    |    *    |    *    |    *    |    *    |    *    |    *    | 
            Seq: CTTCCCTTGGTGCAATGGAcAGCACAACTGAGTCCTAATCAGTAAATAGAGGTTCAACTCCTCTAGGGAAGG
            Str: >>>>>>>..>>>>........>>>>.......>>>>.......<<<<<<<<<<<<.


            Candidate tRNAs not verified by Cove analysis:
            >Your-seq.t1 (Fp Hit: 199996-200090, 95 bp, Src: Eu) 
            ATCACCTTACCCTTTGCTTTAGTGCAAGCATTTGCTGTGATCCAAATTGCTACTAATGCA
            GGCACTGGTTCAAGTCCGATTAGTTTAGCTAATAG
            >Your-seq.t2 (Fp Hit: 222067-222172, 106 bp, Src: Eu) 
            CCTTTTAAAGGCTTTGGTTTAACTGGTAATGCCCCTACTGATTGGAATGAGATCAAAGGT
            AAAGTTCCAGTAGAAGTAGTTCAATCCCCCCATTCCCCCAACCTCT
            >Your-seq.t36 (Fp Hit: 99482-99280, 203 bp, Src: Eu) 
            TTTTTTAACAATGCTACTGTAGTTGGTAAAGCAATTGCAGTAGTACCAAAAGCACCAGAT
            ATTGACAAGCAGATAGTAGTTATTGTACTTAATTTCATATGTTAAGAGAAAAGTTATGAA
            TTTATAAAGAGAGCAAACTTAAACAGCCAATAAGGATCTTATTAGTTGTTGATAAGTTCA
            AATCAAAAGGTTTTATCTCTAAA

  	    </pre>
	</td>
      </tr>
    </table>

    <h2>Command line arguments</h2>
    <table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
      <tr bgcolor="#FFFFCC">
	<th align="left">Qualifier</th>
	<th align="left">Type</th>
	<th align="left">Description</th>
	<th align="left">Allowed values</th>
	<th align="left">Default</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-seqall]<br>(Parameter 1)</td>
	<td>seqall</td>
	<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
	<td>Readable sequence(s)</td>
	<td><b>Required</b></td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>[-outfile]<br>(Parameter 2)</td>
	<td>outfile</td>
	<td>Output file name</td>
	<td>Output file</td>
	<td><i>&lt;*&gt;</i>.ktrnascan_se</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Additional (Optional) qualifiers</th>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-title</td>
	<td>string</td>
	<td>title of your job</td>
	<td>Any string</td>
	<td>&nbsp;</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-mode</td>
	<td>string</td>
	<td>Search mode. Search mode. 'Default', 'Cove only (very slow)', 'tRNAscan only', 'EufindtRNA only', 'EufindtRNA -> Cove' or 'tRNAscan -> Cove'</td>
	<td>Any string</td>
	<td>Default</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-source</td>
	<td>string</td>
	<td>Training data set. 'Mixed (general tRNA model)', 'Eukaryotic', 'Bacterial', 'Archaeal', 'Mito/Chloroplast' or 'Nematode Mito'</td>
	<td>Any string</td>
	<td>Mixed (general tRNA model)</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-gcode</td>
	<td>string</td>
	<td>Genetic Code for tRNA Isotype Prediction. 'Universal', 'Ciliate Nuclear', 'Vertebrate Mito', 'Invertebrate Mito', 'Yeast Mito', 'Echinoderm Mito' or 'Mold & Protozoan Mito'</td>
	<td>Any string</td>
	<td>Universal</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-pesudogene</td>
	<td>boolean</td>
	<td>Disable pseudo gene checking</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-origin</td>
	<td>boolean</td>
	<td>Show origin of first-pass hits</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-ace</td>
	<td>boolean</td>
	<td>Display results in ACeDB format</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-codons</td>
	<td>boolean</td>
	<td>Show codons instead of tRNA anticodons</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-fpos</td>
	<td>boolean</td>
	<td>Show false positives from tRNAscan/EufindtRNA</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-breakdown</td>
	<td>boolean</td>
	<td>Show primary and secondary structure components to Cove scores</td>
	<td>Boolean value Yes/No</td>
	<td>No</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-covescore</td>
	<td>string</td>
	<td>Cove score cutoff</td>
	<td>Any string</td>
	<td>&nbsp;</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-euparams</td>
	<td>string</td>
	<td>EufindtRNA search parameters. 'Default', 'Relaxed' or 'Strict (Pavesi params)'</td>
	<td>Any string</td>
	<td>Default</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<td>-euscore</td>
	<td>string</td>
	<td>Intermediate score cutoff</td>
	<td>Any string</td>
	<td>&nbsp;</td>
      </tr>

      <tr bgcolor="#FFFFCC">
	<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
      </tr>

      <tr>
	<td colspan=5>(none)</td>
      </tr>
    </table>

    <h2>Input file format</h2>
    ktrnascan-se can use any genome sequence data that seqret can read.

    <h2>Output file format</h2>
    The output from ktrnascan_se is general output format of tRNAscan-SE.

    <h2>Data files</h2>
    None.

    <h2>Notes</h2>
    None.

    <h2>References</h2>
    <pre>
      Lowe, T.M., and Eddy, S.R., tRNAscan-SE: a program for improved detection
      of transfer RNA genes in genomic sequence, Nucleic Acids Research, 25, 955-964.
    </pre>

    <h2>Warnings</h2>
    None.

    <h2>Diagnostic Error Messages</h2>
    None.

    <h2>Exit status</h2>
    It always exits with a status of 0.

    <h2>Known bugs</h2>
    None.

    <h2>See also</h2>
    <table border cellpadding=4 bgcolor="#FFFFF0">
      <tr>
	<th>Program name</th>
	<th>Description</th>
      </tr>

      <tr>
	<td><a href="kgenemarkhmm.html">kgenemarkhmm</a></td>
	<td>Finds genes in prokaryotic genomes using GeneMarkhmm</td>
      </tr>

      <tr>
	<td><a href="kglimmer.html">kglimmer</a></td>
	<td>Finds genes in prokaryotic genomes using Glimmer</td>
      </tr>

      <tr>
	<td><a href="seqret.html">seqret</a></td>
	<td>Reads and writes (returns) sequences</td>
      </tr>
    </table>

    <h2>Author(s)</h2>
    Kazuki Oshita (cory @ g-language.org)<br />
    Institute for Advanced Biosciences, Keio University<br />
    252-8520 Japan

    <h2>History</h2>
    2009 - Written by Kazuki Oshita<br />
    Mar 2013 - Update document by Kazuki Oshita

    <h2>Target users</h2>
    This program is intended to be used by everyone and everything, from naive users to embedded scripts.

    <h2>Comments</h2>
    None.

  </BODY>
</HTML>
